### ISSUE TITLE
Error when trying to build package from repos/ASP.

### ISSUE THREAD

**Issue by micwoj92** [Kwpolska/pkgbuilder#69]
```
pkgbuilder --debug python
[INFO   ] :pkgbuilder: *** PKGBUILDer v4.3.1
[DEBUG  ] :pkgbuilder: *** debug output on.
[INFO   ] :pkgbuilder: Arguments parsed.  {'pkgnames': ['python'], 'pac': False, 'fetch': False, 'userfetch': None, 'info': False, 'search': False, 'upgrade': False, 'finst': False, 'runtx': False, 'clean': False, 'noclean': False, 'colors': False, 'nocolors': False, 'debug': True, 'nodebug': False, 'edit_pkgbuild': False, 'noedit_pkgbuild': False, 'depcheck': False, 'nodepcheck': False, 'vcsupgrade': False, 'novcsupgrade': False, 'validation': False, 'novalidation': False, 'pkginst': False, 'nopkginst': False, 'pgpcheck': False, 'nopgpcheck': False, 'confirm': False, 'noconfirm': False, 'shallowclone': False, 'deepclone': False, 'ignorelist': None, 'pacupd': False, 'nopac': False, 'nofetch': False}
[INFO   ] :pkgbuilder: Starting build...
[INFO   ] :pkgbuilder: Building python
[DEBUG  ] :urllib3.connectionpool: Starting new HTTPS connection (1): aur.archlinux.org:443
[DEBUG  ] :urllib3.connectionpool: https://aur.archlinux.org:443 "GET /rpc/?v=5&type=multiinfo&arg%5B%5D=python HTTP/1.1" 200 None
[INFO   ] :pkgbuilder: python not found in the AUR, checking in repositories
Traceback (most recent call last):
  File "/usr/lib/python3.10/site-packages/pkgbuilder/build.py", line 445, in build_runner
    pkg = pkgbuilder.utils.info([pkgname])[0]
IndexError: list index out of range

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/bin/pkgbuilder", line 33, in <module>
    sys.exit(load_entry_point('pkgbuilder==4.3.1', 'console_scripts', 'pkgbuilder')())
  File "/usr/lib/python3.10/site-packages/pkgbuilder/__main__.py", line 356, in main
    out = pkgbuilder.build.auto_build(pkgname, DS.depcheck,
  File "/usr/lib/python3.10/site-packages/pkgbuilder/build.py", line 56, in auto_build
    build_result = build_runner(pkgname, performdepcheck, pkginstall, pkgbuild_edit)
  File "/usr/lib/python3.10/site-packages/pkgbuilder/build.py", line 455, in build_runner
    pkg = pkgbuilder.package.ABSPackage.from_pyalpm(abspkg)
  File "/usr/lib/python3.10/site-packages/pkgbuilder/package.py", line 166, in from_pyalpm
    setattr(p, i, getattr(alpmpkg, i))
AttributeError: 'alpm.Package' object has no attribute 'deltas'
```

**Comment by Kwpolska** [comment]
This should be fixed in PKGBUILDer v4.3.2 (and I hope I didn’t mess up the release process, it’s a little rusty…)

**Comment by micwoj92** [comment]
Thanks for quick fix
